ONSdigital/babbage
# Disallow maxage endpoint when legacy cache api is enabled

### What
Update /maxage endpoint to return 410 (Gone) when legacy cache api is enabled. Removed "LegacyCacheProxyMaxAge" class as we no longer forward requests to the proxy from maxage endpoint

### How to review
1.  Set ENABLE_LEGACY_CACHE_API to "Y"
2. Run babbage: ./run.sh
3. Try hitting a /maxge endpoint e.g./maxage?key=pass&uri=/economy/environmentalaccounts/bulletins/ukenvironmentalaccounts/latest
4. Response should be 410 and message should be "MaxAge endpoint is no longer available within Babbage"

### Who can review
Anyone from ONS
